hack/common.sh

```diff
@@ -1,7 +1,11 @@
 #!/bin/bash
 
-# Location of the manifests file
-MANIFEST_LOC=bundle/
+# Set location of the manifests file
+if [ -z ${COMMUNITY:-} ]; then
+  MANIFEST_LOC=bundle/
+else
+  MANIFEST_LOC="$ARTIFACT_DIR"
+fi
```

Comment:
+1 to Jose's code structure suggestion

note that `-z` checks if a string is null (that is, has zero length) whereas `-n` checks that a string is _not_ null

```suggestion
# set default location of the manifests file
MANIFEST_LOC="$ARTIFACT_DIR"
# update for community 
if [ -n ${COMMUNITY:-} && "$COMMUNITY" = "true" ]; then
  MANIFEST_LOC=bundle/ 
fi
```

Reply:
How is `-n ${COMMUNITY:-}` ever going to evaluate to false?
`${COMMUNITY:-}` by definition cannot be unset by

Reply:
@sebsoto `-n ${COMMUNITY:-}` would evaluate to false if `COMMUNITY=""` (empty string) or the variable is not set (tested this by running `unset COMMUNITY` before the conditional). Since we default to `COMMUNITY="false"` in CI config it won't ever be false but it the check doesn't hurt imo.

Reply:
Got it. Needs quotes around `${COMMUNITY:-}` though.
Instead of putting it into the conditional,  i suggest doing `COMMUNITY=${COMMUNITY:-false}

Reply:
so overall the check looks like this: 
```
MANIFEST_LOC=bundle/
COMMUNITY=${COMMUNITY:-false}
if [ "$COMMUNITY" = "true" ]; then
  MANIFEST_LOC="$ARTIFACT_DIR"
fi
```